apps/src/gamelab/GameLab.js

```diff
@@ -91,6 +101,23 @@ GameLab.prototype.init = function (config) {
     }
   };
 
+  // Override p5.loadImage so we can modify the URL path param
+  if (!GameLab.baseP5loadImage) {
+    GameLab.baseP5loadImage = window.p5.prototype.loadImage;
+    window.p5.prototype.loadImage = function (path, successCallback, failureCallback) {
+      if (ABSOLUTE_REGEXP.test(path)) {
+        // We want to be able to handle the case where our filename contains a
+        // space, i.e. "www.example.com/images/foo bar.png", even though this is a
+        // technically invalid URL. encodeURIComponent will replace space with %20
+        // for us, but as soon as it's decoded, we again have an invalid URL. For
+        // this reason we first replace space with %20 ourselves, such that we now
+        // have a valid URL, and then call encodeURIComponent on the result.
+        path = MEDIA_PROXY + encodeURIComponent(path.replace(' ', '%20'));
```

Comment:
I _think_ this only replaces a single space in the path.  You need to make it a global replacement:

```javascript
path.replace(/ /g, '%20')
```